pkg/virt-controller/services/vm.go

```diff
@@ -192,13 +192,13 @@ func (v *vmService) StartMigration(migration *corev1.Migration, vm *corev1.VM, s
 	if err != nil {
 		return err
 	}
-	_, err = v.KubeCli.CoreV1().Pods(v1.NamespaceDefault).Create(job)
+	_, err = v.KubeCli.CoreV1().Pods(migration.GetObjectMeta().GetNamespace()).Create(job)
 	return err
 }
 
 func (v *vmService) GetMigrationJob(migration *corev1.Migration) (*v1.Pod, bool, error) {
 	selector := migrationJobSelector(migration)
-	podList, err := v.KubeCli.CoreV1().Pods(v1.NamespaceDefault).List(selector)
+	podList, err := v.KubeCli.CoreV1().Pods(migration.ObjectMeta.Namespace).List(selector)
```

Comment:
`obj.ObjectMeta` accesses the metadata through the public field, whereas `obj.GetObjectMeta()` accesses it through a methode defined in the `runtime.Object` interface. We were not really strict on how we want to access it, but I think that accessing it through the method, which is defined to implement the interface is nicer and I normally try to do it that way. On the other hand, accessing it through the public field is easier to read ...